.github/release-please-config.app.json

```diff
@@ -0,0 +1,20 @@
+{
+  "bootstrap-sha": "dc32d0249bde99a124787baa40740bd78e9de3a3",
```

Comment:
What's the purpose of this?

Reply:
the release-please tooling walks the commits backwards to find the previous "release".  When creating the first release, it will walk backwards through all commits in the repo, but this ends up exhausting the github graphql token rate limiter.  This sha points to the earliest commit it will walk backwards to.

Reply:
Cool, thanks. I was looking for it in the docs, but couldn't find it 🤷‍♂️

Reply:
it's kinda hidden in /docs https://github.com/googleapis/release-please/blob/master/docs/manifest-releaser.md#starting-commit